src/main/Controller.java

```diff
@@ -57,30 +57,14 @@ public String selectAction() {
 		return response;
 	}
 	
-	public String searchByKeyword(String keywordInput) {
-		//call the actual search method here 
-		RestAssured.baseURI = "http://webknox.com/api/recipes";
-		String apiKey = "48ec8d46512e4c03a4d2c18b015d64af"; //Aidan API key
-		
-		RequestSpecification httpRequest = RestAssured.given();
-		Response response = httpRequest.request(Method.GET, "/complexSearch?apiKey=" + apiKey + "&query=" + keywordInput);
-		
-		String responseBody = response.getBody().asString();
-		System.out.println("Response Body is =>  " + responseBody);
-		
-		return responseBody;
+	public String callSearchByKeyword(String keywordInput) { 

```

Comment:
This doesn't need to be its own method. What you can do instead is take what you have on line 61 and put it in the constructor and then just call searcher.searchByKeyword(keywordInput) in the case 2 block instead of writing a new method and instantiating APIController twice. I would also rename searcher to aPIController or something along those lines.

Reply:
Hey I tried to update this in the last commit; could you check it out and make sure it looks good?